cli/src/main/java/com/liferay/blade/cli/command/CreateCommand.java

```diff
@@ -299,6 +305,22 @@ protected ProjectTemplatesArgs getProjectTemplateArgs(
 		properties.put("setService", createArgs.getService());
 		properties.put("setViewType", createArgs.getViewType());
 
+		WorkspaceProvider workspaceProvider = getBladeCLI().getWorkspaceProvider(new File(createArgs.getBase()));
+
+		try {
+			if (workspaceProvider != null) {
+				File workspaceLocation = workspaceProvider.getWorkspaceDir(getBladeCLI());
+
+				if (workspaceLocation != null) {
+					properties.put("setModulesLocation", _getDefaultModulesDir().toString());
```

Comment:
Having to set these properties like this means our ProjectTemplateArgs ext api is broken.  If we have to resort to stringify these methods, then we might as well just pass hashMaps all around and remove the arg extensions api.

Reply:
Agreed that this isn't ideal, and I think eventually we can find a better solution than passing in the method names / values as properties to Project Templates.